src/components/styles/Calculator.module.css

```diff
@@ -0,0 +1,12 @@
+.body {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 4% 10%;
+}
+
+@media screen and (min-width: 768px) {
+  .body {
+    grid-template-columns: 1fr 2fr;
+    padding: 4% 10%;
+  }
+}
```

Comment:
- [ ] - [x] Sincerely I really commend your project, the styling is great! But according to the project requirement, you're suppose to use one of the **React basic styling** methods. Please kindly implement this for the four other modules as well (`Header.module.css`, `Home.module.css`,  `Links.module.css` and `Quotes.module.css`), you can do it. 👍 

<img width="565" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/228375056-f99b00f1-0879-41bd-8bf9-fe20f7a0a510.png">